src/license/mapping.xml

```diff
@@ -42,6 +42,31 @@
         <artifactId>org.osgi.core</artifactId>
         <license>Apache License, Version 2.0</license>
     </artifact>
+    <artifact>
+        <groupId>org.bouncycastle</groupId>
+        <artifactId>bcmail-jdk15on</artifactId>
+        <license>Bouncy Castle/MIT License</license>
+    </artifact>
+    <artifact>
+        <groupId>org.bouncycastle</groupId>
+        <artifactId>bcpkix-jdk15on</artifactId>
+        <license>Bouncy Castle/MIT License</license>
+    </artifact>
+    <artifact>
+        <groupId>org.bouncycastle</groupId>
+        <artifactId>bcprov-jdk15on</artifactId>
+        <license>Bouncy Castle/MIT License</license>
+    </artifact>
+    <artifact>
+        <groupId>com.google.code.gson</groupId>
+        <artifactId>gson</artifactId>
+        <license>Apache License, Version 2.0</license>
+    </artifact>
+    <artifact>
+        <groupId>io.jsonwebtoken</groupId>
```

Comment:
DId this one not get picked up automatically?
If not let me know, and I'll make sure to get that fixed on the JJWT side (for future releases)

Reply:
yes, I had to manually add this mapping in this file. Otherwise mvn build would fail.